crates/ruff_python_index/src/indexer.rs

```diff
@@ -28,19 +22,18 @@ pub struct Indexer {
 }
 
 impl Indexer {
-    pub fn from_tokens(tokens: &[LexResult], locator: &Locator) -> Self {
+    pub fn from_program(program: &Program<ModModule>, locator: &Locator<'_>) -> Self {
```

Comment:
Same as for `Stylist`. Should this just accept the `Tokens` instead of the entire program?